tests/integration_tests/performance/test_boottime.py

```diff
@@ -57,6 +58,10 @@ def test_multiple_microvm_boottime_with_network(
         network_config
 ):
     """Check guest boottime with network when spawning multiple microvms."""
+    if platform.machine().lower() == 'arm':
+        pytest.skip("Skiped because it is failing randomly.\
+                     Needs investigations.")
+
```

Comment:
I vote for actually removing this test (PR https://github.com/firecracker-microvm/firecracker/pull/1664)

Reply:
Does it fail on x86_64 as well or only on arm?

Reply:
For 10 microVMs it *very rarely* fails on x86_64 and *very often* on arm.
For a higher number of microVMs it fails on both.